aep/general/0133/aep.md.j2

```diff
@@ -0,0 +1,246 @@
+# Create
+
+In REST APIs, it is customary to make a `POST` request to a collection's URI
+(for example, `/v1/publishers/{publisher}/books`) in order to create a new
+resource within that collection.
+
+Resource-oriented design (AEP-121) honors this pattern through the `Create`
+method. These RPCs accept the parent collection and the resource to create (and
+potentially some other parameters), and return the created resource.
+
+## Guidance
+
+APIs **should** generally provide a create method for resources unless it is
+not valuable for users to do so. The purpose of the create method is to create
+a new resource in an already-existing collection.
+
+Create methods are specified using the following pattern:
+
+- The response **must** be the resource itself. There is no separate response
+  schema.
+  - The response **should** include the fully-populated resource, and **must**
+    include any fields that were provided unless they are input only (see
+    AEP-203).
+  - If the create RPC is [long-running](#long-running-create), the response
+    message **must** be an `Operation` for which the type of the `response`
+    field is the resource itself.
+- The HTTP verb **must** be `POST`.
+- The collection where the resource is being added **should** map to the URI
+  path.
+  - The collection's parent resource **should** be called `parent`, and
+    **should** be the only variable in the URI path.
+  - The collection identifier (`books` in the above example) **must** be a
+    literal string.
+- If the API is operating on the [management plane][], the operation should
+  have [strong consistency][]: the completion of a create operation **must**
```

Comment:
strong consistency link is broken.

Reply:
I actually don't know what this is supposed to link to.  The link is broken at https://google.aip.dev/133 as well.

I'm just going to remove this bullet for now - we can always add it back later after we merge the planes AEP.

Reply:
I'd rather keep it actually - it's a good note and I don't want to forget re-adding it. It's also good guidance: eventual consistency is confusing to most users at first blush.

If the management plane reference is an issue, I'm fine with just making this a generic "SHOULD"

Here's the place it should link to: https://aep.dev/121#strong-consistency

Reply:
Thanks for the actual link target. :) 

Regarding whether it's good guidance in general...I agree strong consistency is a nice thing to have, of course.  But if someone has a system architecture that doesn't allow for strong consistency, are they ever going to change that because of a **should** in the AEPs?  "Strong consistency is preferable" seems like A) a truism; B) more about system architecture than API design; and C) somewhat of a blinkered recommendation, as if the AEPs are oblivious to the system design challenges people might be facing when this guidance would apply.

WDYT?  If that's not persuasive, thoughts of filing an issue to revisit this?

Reply:
I think there's plenty of things that are shoulds in the AEPs that one wouldn't change because it's in there - so I don't think that's a good reason to exclude this one. Similar to how an adopter of the AEP won't change their HTTP method to POST because the AEPs say so.

regardless of the difficulty of implementing it on the backend, good guidance is good guidance - and it's something people should consider, especially when authoring an API from scratch. The reason it's a *SHOULD* (I'd love to live in a world where it could be a *MUST*) is because the reality is some systems are hard to change, or is in direct conflict with backend behavior that is more desirable. 

Since this is already in the AIP, I'd rather see it merged in and file an issue to change it away from AIP guidance. To make this more of a principled decision: I'd agree with defaulting to AIP guidance and filing issues with the suggested change (thereby putting the onus on the person who disagrees with upstream guidance to argue for it).

Reply:
Sure, leaving it in and filing an issue to bias towards not blocking PRs on controversial modifications.